adminrestrict/tests.py

```diff
@@ -90,6 +90,15 @@ def test_custom_denied_msg(self):
             self.assertEqual(resp.content, DENIED_MSG)
         a.delete()
 
+    def test_custom_denied_template(self):
+        DENIED_TEMPLATE = b"403.html!"
+        a = AllowedIP.objects.create(ip_address="16*")
+        with self.settings(ADMINRESTRICT_TEMPLATE=DENIED_TEMPLATE):
+            resp = self.client.post("/admin/", data={'username':"foo", 'password':"bar"}, follow=True)
```

Comment:
This is a good test for the POST version.

Let's add a test to GET the page and that will verify line 225 as well.

Reply:
I see the test being done on line 30:

def test_disallow_get(self):
        a = AllowedIP.objects.create(ip_address="10.10.0.1")
        with self.settings(ADMINRESTRICT_BLOCK_GET=True):
            resp = self.client.get("/admin/")
            self.assertEqual(resp.status_code, 403)
        a.delete()

Reply:
Line 30 is part of a different test to verify a different feature.

What I mean is we can test the custom template for GET requests too.

Reply:
Added the tests. 😀